docs/gettingstarted/structuring.md

```diff
@@ -25,6 +25,69 @@ What is `@Mod`?
 
 This is an annotation indicating to the Forge Mod Loader that the class is a Mod entry point. It contains various metadata about the mod. It also designates the class that will receive `@EventHandler` events. More information can be found at... (Coming Soon)
 
+All components of @Mod() are written inside the parantheses and chained together. There's only ever a single @Mod in your entire mod, sitting on top of your entry class.
```

Comment:
Do I not, though? Where does the java tutorial end and the forge tutorial start?
Additionally: What would the purpose of multiple `@Mod` annotations be? Do they combine behind the scenes? 
This is not immediately apparent to whoever is new to modding. (I presume this description is aiming to help both the beginner and immediate modder.)